python-lib/googlesheets.py

```diff
@@ -79,3 +81,6 @@ def get_spreadsheets(self, document_id, tab_id=None):
                 if error_status == 'NOT_FOUND':
                     raise Exception("Trying to open non-existent spreadsheet document. Verify the document id exists (%s)." % document_id)
             raise Exception("The Google API returned an error: %s" % error)
+
+    def get_spreadsheet_title(self, document_id):
+        return self.client.open_by_key(document_id).title
```

Comment:
Maybe we should add `try`/`catch` here just like the previous function?

Reply:
Done in [607299a](https://github.com/dataiku/dss-plugin-googlesheets/pull/7/commits/607299a6a30aba743cacbbee9c55ee6c2668c44e)